mobile/examples/React_Native/ort-rn-basic-usage/README.md

```diff
@@ -0,0 +1,109 @@
+# ONNX Runtime React Native Expo Example Application
+
+This is a [React Native](https://reactnative.dev/docs/getting-started) application for [ONNX Runtime](https://github.com/microsoft/onnxruntime) via [Expo](https://docs.expo.dev/) platform. The demo app demonstrates how to accomplish simple tasks such as loading onnx models and creating inference sessions, etc. in a react native expo application.
+
+## Prerequisites
+
+1. Install [Node.js](https://nodejs.org/en)
+2. Install [Expo-CLI](https://docs.expo.dev/more/expo-cli/)
+    ```sh
+    npm install -g expo-cli
+    ```
+3. Install [Yarn](https://classic.yarnpkg.com/en/docs/install#mac-stable) (Recommended)
+    ```sh
+    npm install -g yarn
+    ```
+**NOTE:**
+   For creating a new react native expo project from scratch, refer to: https://docs.expo.dev/get-started/create-a-project/
+
+## Set up
```

Comment:
The files that are checked in have had all these 'Set up' steps done right? 

If so, this should be called something like 'Steps that were done to add onnxruntime-react-native to the example expo app' so the user can replicate that in their own app.

We should add an item between Prerequisites and this for 'Validate your React Native environment' where we ask them to run the checked in app as-is (maybe move the 'Build and run' info to this new item)
-If the checked-in app fails for us, it's probably an ORT issue
  - ideally we have a CI running the app so we don't have to manually validate this
-If the app fails for them but not us it's probably their RN env. 
-If the app works for them, it's most likely something in their code/setup.

Reply:
> The files that are checked in have had all these 'Set up' steps done right?

 The user who clone this app will still have to run `yarn install` and `expo install` the react native npm package. And yes for step `3.` and `4.` are by default done for this sample. put a comment in those steps already.

will see if a CI is possible.

Reply:
updated to add a new section as suggested.. hopefully it's clearer now.